### PR TITLE
Update Airbase to version 129

### PR DESCRIPTION
**Pull request by nineinchnick**
## Description

There's a new Checkstyle rule:
* Require package name to match the source directory

## Non-technical explanation

n/a

## Release notes

(x) This is not user-visible and no release notes are required.
( ) Release notes are required, please propose a release note for me.
( ) Release notes are required, with the following suggested text:

### CODE REVIEW

**Review by wendigo — APPROVED** [review]
Thank you!

File: testing/trino-product-tests/pom.xml [hunk 1]
```diff
@@ -340,6 +340,14 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-checkstyle-plugin</artifactId>
+                <configuration>
+                    <!-- required to inject a custom redirection handler into the JDBC driver -->
+                    <excludes>**/TestingRedirectHandlerInjector.java</excludes>
```
Comment (findepi) [inline testing/trino-product-tests/pom.xml:348]: I don't believe the exclusion is correct.
The TestingRedirectHandlerInjector class simply resides in a wrong folder.

Comment (findepi) [reply testing/trino-product-tests/pom.xml:348]: it's a test code's hook, not API

Comment (nineinchnick) [reply testing/trino-product-tests/pom.xml:348]: Should I move it to `client/trino-jdbc/src/main` or `client/trino-jdbc/src/test`?

Comment (nineinchnick) [reply testing/trino-product-tests/pom.xml]: Oooh 🤦

**Comment by findepi** [comment]
CI https://github.com/trinodb/trino/issues/14170

**Pull request merged**